charts/incubator/notifiarr/values.yaml

```diff
@@ -0,0 +1,44 @@
+image:
+  repository: tccr.io/truecharts/notifiarr
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:913c71f5e325dd9af53c9e96f3d5d7d1b2d058a41f8253489320a4c69fbb0a44
+
+service:
+  main:
+    ports:
+      main:
+        port: 5454
+        protocol: http
+        targetPort: 5454
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            TMPDIR: /tmpdir
+            DN_API_KEY: "CHANGEME"
+            DN_UPSTREAMS_0: "172.16.0.0/16"
+            DN_UI_PASSWORD: "username:9CharacterPassword"
+
+persistence:
+  config:
+    enabled: true
+    mountPath: /config
+  utmp:
+    enabled: true
+    mountPath: /var/run/utmp
+  machine-id:
+    enabled: true
+    mountPath: /etc/machine-id
+    readOnly: true
+  tmpdir:
+    enabled: true
+    type: emptyDir
+    mountPath: /tmpdir
```

Comment:
I don't see this mount in the upstream repository?

Reply:
This one works with the TMPDIR env var. It's an optional variable used to set the temporary directory for checking *arr application databases for corruption, so by setting it we can do those checks in memory.